test/e2e/info_test.go

```diff
@@ -49,4 +53,32 @@ var _ = Describe("Podman Info", func() {
 		Expect(session.ExitCode()).To(Equal(0))
 		Expect(session.IsJSONOutputValid()).To(BeTrue())
 	})
+	It("podman info rootless storage path", func() {
+		if !rootless.IsRootless() {
+			Skip("test rootless_storage_path for rootless")
+		}
+		SkipIfRemote()
+		configPath := "/etc/containers/storage.conf"
```

Comment:
I'm really confused. In rootless, doesn't podman use `$HOME/.config/containers/storage.conf`? Shouldn't this test be using that?